src/main/java/com/joejoe2/chat/service/channel/GroupChannelServiceImpl.java

```diff
@@ -120,7 +120,7 @@ else if (subscriber instanceof WebSocketSession)
   public SseEmitter subscribe(String fromUserId) throws UserDoesNotExist {
     userRepository
         .findById(uuidValidator.validate(fromUserId))
-        .orElseThrow(() -> new UserDoesNotExist("user is not exist !"));
+        .orElseThrow(() -> new UserDoesNotExist(fromUserId));
```

Comment:
Regarding parameters it needs to look like the following:
```java
new UserDoesNotExist("User with an Id=" + fromUserId + " doesn't exist")
```

same applies t the channel.